mcs/class/Microsoft.Build.Engine/Test/Microsoft.Build.BuildEngine/ProjectTest.cs

```diff
@@ -1301,6 +1301,34 @@ public void TestBuildProjectFileXml8 ()
 		}
 
 		[Test]
+		public void TestBatchedMetadataEmptyBatch () {
```

Comment:
It fixes the mistake in Haakon Sporsheim's realisation of ItemDefinitionGroup.
Pls see change of Microsoft.Build.BuildEngine/Project.cs in ffa32d6